canonical/canonical.com
# Update favicons

## Done

- Updated favicons as per request

Note: It is still not clear to me which asset is the correct replacement for a given favicon.  

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- See that the favicons have been updated

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4615
